src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -171,7 +169,7 @@ public String toFormattedString() {
 			from = ChatColor.GRAY + "   " + from + "\n";
 
 		return
-			String.format(lineInfoMsg, String.valueOf(node.getLine()), c.getFileName()) + // String.valueOf is to convert the line number (int) to a String
+			String.format(lineInfoMsg, node.getLine(), c.getFileName()) +
```

Comment:
Is this safe? Doesn't the lang message need to be changed to not use %s as the format specifier now?

Reply:
I tested it to make sure and it works fine, so I think java's smart enough to convert it.